static/compiler.js

```diff
@@ -245,6 +252,13 @@ define(function (require) {
             this.optButton.prop("disabled", true);
         }, this));
 
+        this.astButton.click(_.bind(function () {
```

Comment:
Would be nice to support dragging here. The other buttons (e.g. clone compiler and opt view) support dragging to create. (see lines 236ish for this).